staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -63,7 +63,13 @@ type Attributes interface {
 	// "podsecuritypolicy" is the name of the plugin, "admission.k8s.io" is the name of the organization, "admit-policy" is the key name.
 	// An error is returned if the format of key is invalid. When trying to overwrite annotation with a new value, an error is returned.
 	// Both ValidationInterface and MutationInterface are allowed to add Annotations.
+	// By default, an annotation gets logged into audit event if the request's audit level is greater or
+	// equal to Metadata.
 	AddAnnotation(key, value string) error
+
+	// AddAnnotationWithLevel sets annotation according to key-value pair with additional intended audit level.
+	// An Annotation gets logged into audit event if the request's audit level is greater or equal to the
+	// intended audit level.
 	AddAnnotationWithLevel(key, value string, level auditinternal.Level) error
```

Comment:
do we need two funcs?

Reply:
otherwise we need to update places invoking `AddAnnotation()`, and they need to import `auditinternal` package. I'd like those places (test code, some admission controller package) to be unaware of `auditinternal`